aspnet/mvc/views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImports.cshtml

```diff
@@ -1,11 +1,11 @@
 ﻿@using AuthoringTagHelpers
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.EmailTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.EmailTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.WebsiteInformationTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.ConditionTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerHttpTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerWwwTagHelper, AuthoringTagHelpers"
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.BoldTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.BoldTagHelper, AuthoringTagHelpers"
```

Comment:
I'm pretty sure those NS changes will bring in early (and wrong) TH so the code won't work. I develop the TH in stages and use the NS to prevent collisions.